### DISCUSSION TITLE
Documentation does not list configuration file location

### DISCUSSION THREAD

**Discussion by Baa14453** [rhubarbgroup/.github#31] [Support (ROC)]
## Description
The [configuration page](https://github.com/rhubarbgroup/redis-cache/#configuration) has a list of handy config options, but it says if you don't know how to use them you should check out the [installation instructions](https://github.com/rhubarbgroup/redis-cache/blob/develop/INSTALL.md#installing-redis-object-cache)

However, in the installation instructions under [Configuring the plugin](https://github.com/rhubarbgroup/redis-cache/#configuration) it says:

> By default the object cache will connect to Redis Server over TCP at 127.0.0.1:6379 and use database 0, if you see Status: Not connected either ask your hosting provider for assistance, or [configure the connection yourself](https://github.com/rhubarbgroup/redis-cache/#configuration).

The documentation just goes in a loop, I don't think it actually says anywhere how you can edit the config file yourself or where it is located.

## Expected Behavior
The documentation should say where the config file can be found.

## Actual Behavior
The documentation loops back on itself.

## Possible Fix
Include the location of the config file somewhere.

## Steps to Reproduce
1. Visit the [configuration page](https://github.com/rhubarbgroup/redis-cache/#configuration)
2. Visit the [installation instructions](https://github.com/rhubarbgroup/redis-cache/blob/develop/INSTALL.md#installing-redis-object-cache)
3. Visit the [configuration page](https://github.com/rhubarbgroup/redis-cache/#configuration)
4. Visit the [installation instructions](https://github.com/rhubarbgroup/redis-cache/blob/develop/INSTALL.md#installing-redis-object-cache)
5. etc.

**Comment by tillkruss** [comment]
Do you want to open a PR with the suggested change(s)?

**Reply by Baa14453** [reply]
Yeah I can do but to be honest I'm still not sure where the config file is

**Comment by curtismchale** [comment]
@Baa14453 the configuration options should be put in `wp-config.php` which it does list later in the Install instructions.
